lua/nvim-treesitter/install.lua

```diff
@@ -76,83 +111,81 @@ local function run_install(cache_folder, package_path, lang, repo)
     }
   }
 
-  iter_cmd(command_list, 1, lang)
+  if with_sync then
+    if iter_cmd_sync(command_list, lang) == true then
+      print('Treesitter parser for '..lang..' has been installed')
+    end
+  else
+    iter_cmd(command_list, 1, lang)
+  end
 end
 
 -- TODO(kyazdani): this should work on windows too
-local function install(...)
-  if fn.has('win32') == 1 then
-    return api.nvim_err_writeln('This command is not available on windows at the moment.')
-  end
+local function install(with_sync)
+  return function (...)
+    if fn.has('win32') == 1 then
+      return api.nvim_err_writeln('This command is not available on windows at the moment.')
+    end
 
-  if fn.executable('git') == 0 then
-    return api.nvim_err_writeln('Git is required on your system to run this command')
-  end
+    if fn.executable('git') == 0 then
+      return api.nvim_err_writeln('Git is required on your system to run this command')
+    end
 
-  local package_path, err = utils.get_package_path()
-  if err then return api.nvim_err_writeln(err) end
+    local package_path, err = utils.get_package_path()
+    if err then return api.nvim_err_writeln(err) end
 
-  local cache_folder, err = utils.get_cache_dir()
-  if err then return api.nvim_err_writeln(err) end
+    local cache_folder, err = utils.get_cache_dir()
+    if err then return api.nvim_err_writeln(err) end
 
-  local languages = { ... }
-  local check_installed = true
-  if ... == 'all' then
-    languages = parsers.available_parsers()
-    check_installed = false
-  end
+    local languages = vim.tbl_flatten({...})
+    local ask_reinstall = true
+    if ... == 'all' then
+      languages = parsers.available_parsers()
+      ask_reinstall = false
+    end
```

Comment:
Can we change that to:
```suggestion
    local languages
    local ask_reinstall
    if ... == 'all' then
      languages = parsers.available_parsers()
      ask_reinstall = false
    else
      languages = vim.tbl_flatten({...})
      ask_reinstall = true
    end
```
 I find it easier to read...